modular_skyrat/modules/cargo/code/items/AFAD.dm

```diff
@@ -0,0 +1,150 @@
+/obj/item/gun/afad
```

Comment:
Do we need to recode this in its entirety? Can it not be a subtype of the medbeam?

Reply:
redone